robin.js

```diff
@@ -16,11 +16,13 @@ module.exports = function (n, ps) {  // n = num players
     ps.push(DUMMY); // so we can match algorithm for even numbers
     n += 1;
   }
-  for (var j = 0; j < n - 1; j += 1) {
+  for (let j = 0; j < n - 1; j += 1) {
     rs[j] = []; // create inner match array for round j
-    for (var i = 0; i < n / 2; i += 1) {
-      if (ps[i] !== DUMMY && ps[n - 1 - i] !== DUMMY) {
-        rs[j].push([ps[i], ps[n - 1 - i]]); // insert pair as a match
+    for (let i = 0; i < n / 2; i += 1) {
+      const o = n - 1 - i;
+      if (ps[i] !== DUMMY && ps[o] !== DUMMY) {
+        const iHome = i === 0 && j % 2 === 1;
+        rs[j].push([iHome ? ps[o] : ps[i], iHome ? ps[i] : ps[o]]); // insert pair as a match - [ away, home ]
```

Comment:
is not `[home, away]` more expected of an ordering?

Reply:
Typically in sports, the home team is listed second as it would be written out like so: `Team 2 @ Team 1`. That's where my logic comes from anyways.

Reply:
Ok, let's go with this then :-)